hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/recon/TestReconWithOzoneManager.java

```diff
@@ -381,16 +382,21 @@ private static OmKeyLocationInfoGroup getOmKeyLocationInfoGroup() {
 
   private long getReconTaskAttributeFromJson(String taskStatusResponse,
                                              String taskName,
-                                             String entityAttribute) {
-    ArrayList<LinkedTreeMap> taskStatusList = new Gson()
-        .fromJson(taskStatusResponse, ArrayList.class);
+                                             String entityAttribute)
+      throws IOException {
+    ObjectMapper objectMapper = new ObjectMapper();
+    ArrayList<LinkedTreeMap> taskStatusList = objectMapper.readValue(
+        taskStatusResponse, new TypeReference<ArrayList<LinkedTreeMap>>() {
```

Comment:
Please change `ArrayList<LinkedTreeMap>` to `List<Map<String, Object>>`.

`LinkedTreeMap` is from GSON.